mountinfo/openat2_linux.go

```diff
@@ -0,0 +1,44 @@
+package mountinfo
+
+import (
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+)
+
+type openHow struct {
+	Flags   uint64
+	Mode    uint64
+	Resolve uint64
+}
+
+const sizeofOpenHow = 0x18
```

Comment:
A minor thing, but this makes me a little nervous -- is it possible to use `unsafe.Sizeof`?

Reply:
1. This is temp POC code.
2. This is generated code, the generator itself (in x/sys/unix) is actually using C sizeof.